.github/workflows/ci.yml

```diff
@@ -61,7 +86,11 @@ jobs:
       - uses: Swatinem/rust-cache@23bce251a8cd2ffc3c1075eaa2367cf899916d84 # v2
       - uses: taiki-e/install-action@cargo-hack
       - uses: taiki-e/install-action@just
+      - uses: taiki-e/install-action@nextest
       - name: Build
         run: just powerset build
-      - name: Test
-        run: just powerset test
+      # We don't run ui_test on the MSRV since compiler output varies by version.
+      - name: Test without ui_test
+        run: just powerset nextest run -E 'not binary(ui_test)'
```

Comment:
Had to use nextest for this because cargo test doesn't provide a way to exclude a binary.